chapters/fa/_toctree.yml

```diff
@@ -15,6 +15,11 @@
   - local: chapter2/2
     title: پشت صحنه خط‌تولید
 
+- title: ۴- به‌اشتراک‌گذاری مدل‌ها و توکِن‌سازها
```

Comment:
I did a quick search for your equivalent of tokenizer, "token-saz" on Google, which is what a newbie reading this word in the course will do, all the content there relates to bitcoin wallets and whatnot.--this is the Persian information bubble we talked about before. Sad side effect of this is also breaking the automatic link btw Tokenizer-related classes and methods in code, which is a privilege you are denying our Persian-speaking readers. I think we should stick with transliteration on this one.

Reply:
Oh sorry, this is my bad; I forgot to apply what we agreed on discussion page, fixed it now.